src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,277 @@
+package duke;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * The main class controlling the chatbot's actions.
+ * This class should be able to keep track of tasks, and
+ * send messages to the user.
+ */
+public class Duke {
+    /**
+     * Common text elements to be reused by the chatbot, such as the chatbot name,
+     * and the horizontal line element.
+     */
+    private static final String name = "JOHNATHAN CENATOR\n";
+    private static final String horizontalLine = "-------------------------------\n";
+
+    /**
+     * Store tasks in a separate class
+     */
+    private static Storage taskStorage = new Storage();
+
+    /**
+     * An enum to track the status of the chatbot
+     * RUNNING, STOPPING, etc.
+     * [To add more if needed]
+     */
+    enum Status {RUNNING, STOPPING}
+
+    /**
+     * Sends a greeting message on startup of the chatbot.
+     */
+    private static void greet() {
+        System.out.print(horizontalLine +
+                "YOU DIDN'T SEE\n" +
+                name +
+                "COMING DID YOU!?\n" +
+                horizontalLine);
+    }
+
+    /**
+     * Sends a departing message on chatbot shutdown.
+     */
+    private static void exit() {
+        System.out.print(horizontalLine +
+                "NOW GET OUTTA HERE!\n" +
+                "RESPECTFULLY,\n" +
+                name +
+                horizontalLine);
+    }
+
+    /**
+     * Repeats the user's input
+     *
+     * @param input the user's text input
+     */
+    private static void echo(String input) {
+        System.out.print(horizontalLine + input + "\n" + horizontalLine);
+    }
+
+    /**
+     * Lists all tasks in the task array
+     */
+    private static void list() {
+        System.out.print(horizontalLine);
+        System.out.print(taskStorage.list());
+        System.out.print(horizontalLine);
+    }
+
+    /**
+     * Appends a task to the task array
+     *
+     * @param task The task inputted by the user
+     */
+    private static void append(Task task) {
+        taskStorage.appendTask(task);
+        System.out.print(horizontalLine + "YOU WANT TO " + task + "?\nSURE, WHATEVER.\n" + horizontalLine);
```

Comment:
Any reason why you decided to keep this in one line instead of 
```java
System.out.print(horizontalLine +
                "YOU DIDN'T SEE\n" +
                name +
                "COMING DID YOU!?\n" +
                horizontalLine);
```
like the above?